Project.toml

```diff
@@ -1,21 +1,21 @@
 name = "GraphicalModelLearning"
 uuid = "73787735-dbe8-5e05-9488-dc58b1b075d0"
-repo = "https://github.com/lanl-ansi/GraphicalModelLearning.jl.git"
 authors = ["Marc Vuffray, Carleton Coffrin"]
+repo = "https://github.com/lanl-ansi/GraphicalModelLearning.jl.git"
 version = "0.1.0"
```

Comment:
Version should be increased to v0.2.0, and a note should be added to the changelog.md